### DISCUSSION TITLE
Seams - More consistent Aligned & Corner?

### DISCUSSION THREAD

**Discussion by Durahl** [supermerill/SuperSlicer#2799] [Q&A] [answered]
Is there a way to achieve a more consistent _and_ favorable _Aligned_ Seam position?

Even with the most recent 2.4.58.2 version - Using _Aligned_ in **SuperSlicer** often enough places them at the IMHO most inappropriate locations like straight down a face instead of a more sensible one like along a sharp corner while _also_ not aligning them at all:
![IMG](https://i.imgur.com/5SEgFRN.png)
**PrusaSlicer** for comparison handles this more favorable in at least the way that it consistently positions them at corners where possible - Though at times still scattered all around the model:
![IMG](https://i.imgur.com/yK9edCb.png)

**On a somewhat unrelated question...**
I used to be able to Sync my SS Settings across my various PCs by copy/pasting the entire Settings Folder _once_ ( found in: `C:\Users\USERNAME\AppData\Roaming\SuperSlicer` ) to a Cloud synchronized folder and then adding _that_ folder to the end of the Shortcut **Target** entry: `--datadir "C:\Users\USERNAME\OneDrive\Documents\SuperSlicer"` but as of recently that seems to be broken as attempting to start SS with that modified Shortcut ends in SS not starting at all - Any idea what changed? Is there may be an even more elegant solution available? 🤔

**Comment by tome9111991** [comment, marked as answer]
you van check the nightly
https://github.com/supermerill/SuperSlicer/actions/runs/2281591910

seams are fixed.

**Comment by Durahl** [comment]
![IMG](https://i.imgur.com/vXce5YF.png)
![ChefsKissFrenchChefGIF](https://user-images.githubusercontent.com/10585968/168193569-94d5d94b-ecaa-4559-bf9f-348bc469d506.gif)
